bbc/simorgh
# add new style to remove focus

Resolves JIRA [1206](https://jira.dev.bbc.co.uk/browse/WSTEAM1-1206)

Overall changes
======
added a new style `heading` thats applied to the `Heading` component which sets the `focus` to `none`.

Testing
======
1. visit a live page `http://localhost:7081/pidgin/live/c7p765ynk9qt`
2. Use the skip to content button 
3. Check that there is no outline on the title

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
